widgets/volume_linux.lua

```diff
@@ -26,8 +26,22 @@ local function worker(format, warg)
         ["off"] = "♩"  -- "M"
     }
 
+    local mixer = nil
+    local device = nil
```

Comment:
using local is always a good idea. The default value of a variable should be nil already.